### DISCUSSION TITLE
Cron job isn't working if Attack challenge mode is enabled

### DISCUSSION THREAD

**Discussion by Naeem-gg** [vercel/community#6022] [Bug Report] [answered]
### Page to Investigate

https://vercel.com/[user]/[project]/settings/security

### Steps to Reproduce

I don't know if this is supposed to be like this or not, But after enabling this Attack Challenge Mode I'm getting 429 Too Many Requests as log.

**Comment by amyegan** [comment]
Thanks for reaching out about this @Naeem-gg. Attack Challenge Mode is very new so I don't know the answer to this one yet. I'm going to check with the engineering team that built the feature, and I'll let you know when I learn more from them. 🙇

**Reply by amyegan** [reply, marked as answer]
Thanks for your patience, everyone. The team confirmed this is expected because every request is automatically challenged when Attack Challenge Mode is enabled. It is meant to temporarily add an extra layer of control when your site is under attack and should be disabled when it is not actively needed. We'll get the docs updated to clarify this.

If you continue to see errors after disabling Attack Challenge Mode, please let me know

**Comment by Amirthananth** [comment]
Facing the same issue. Would be great to have it sorted out at the earliest.

Thanks

**Comment by ginodaloisio** [comment]
Facing the same issue here, hope we get some news on it. 

Thanks

**Comment by nolinnaidoo-agentzero** [comment]
Facing the same issue, this is a major issue us. Eagerly awaiting an update/fix.

**Comment by Amirthananth** [comment]
@amyegan Thanks for the reply. Is there any plans to have the corn job working even if the attack mode is enabled ?

**Reply by Naeem-gg** [reply]
https://github.com/orgs/vercel/discussions/6022#discussioncomment-9383676

**Comment by Naeem-gg** [comment]
> Facing the same issue, this is a major issue us. Eagerly awaiting an update/fix.

https://github.com/orgs/vercel/discussions/6022#discussioncomment-9383676

**Comment by Naeem-gg** [comment]
> Facing the same issue here, hope we get some news on it. 
> 
> Thanks

https://github.com/orgs/vercel/discussions/6022#discussioncomment-9383676
